### PR TITLE
feat: add Dependabot YAML file

### PR DESCRIPTION
**Pull request by liammulh**
By submitting this PR, I am indicating to the Numberscope maintainers that I have read and understood the [contributing guidelines](https://numberscope.colorado.edu/doc/CONTRIBUTING/) and that this PR follows those guidelines to the best of my knowledge. I have also read the [pull request checklist](https://numberscope.colorado.edu/doc/doc/pull-request-checklist/) and followed the instructions therein.

<hr/>

Basically the title. Adds configuration for Dependabot. Note that I have it configured to run weekly, which we might want to change to daily. Resolves https://github.com/numberscope/frontscope/issues/24.

### CODE REVIEW

**Comment by gwhitney** [comment]
Not sure how to test this, so am going to take your word for it as it looks plausible. Merging.
